packages/react-is/README.md

```diff
@@ -0,0 +1,89 @@
+# `react-is`
+
+This package allows you to test arbitrary values and see if they're a particular React type, e.g. React Elements.
+
+## Installation
+
+```sh
+# Yarn
+yarn add react-is
+
+# NPM
+npm install react-is --save
+```
+
+## Usage
+
+### AsyncMode
+
+```js
+import React from "react";
+import { AsyncMode, isAsyncMode, typeOf } from "react-is";
+
+isAsyncMode(<React.unstable_AsyncMode />); // true
+typeOf(<React.unstable_AsyncMode />) === AsyncMode; // true
+```
+
+### Context
+
+```js
+import React from "react";
+import {
+  ContextConsumer,
+  ContextProvider,
+  isContextConsumer,
+  isContextProvider,
+  typeOf
+} from "react-is";
+
+const ThemeContext = React.createContext("blue");
+
+isContextConsumer(<ThemeContext.Consumer />); // true
+isContextProvider(<ThemeContext.Provider />); // true
+typeOf(<ThemeContext.Provider />) === ContextProvider; // true
+typeOf(<ThemeContext.Consumer />) === ContextConsumer; // true
+```
+
+### Element
+
+```js
+import React from "react";
+import { Element, isElement, typeOf } from "react-is";
+
+isElement(<div />); // true
+typeOf(<div />) === Element; // true
+```
+
+### Fragment
+
+```js
+import React from "react";
+import { Fragment, isFragment, typeOf } from "react-is";
```

Comment:
I could see this being confusing (e.g. when you read `Fragment` you assume it's from `react`).

Maybe let's keep it grouped, like before? It just needs to be a namespace import rather than a default one.

```js
import * as ReactIs from 'react-is';
```